### PR TITLE
[13.0][IMP] account_payment_partner: Add Payment mode to Invoices Analysis

### PR DESCRIPTION
**Pull request by victoralmau**
Add Payment mode to Invoices Analysis.

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT30685

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1:

File: account_payment_partner/README.rst [hunk 1]
```diff
@@ -46,6 +46,8 @@ information, but there are some customers that don't want that everyone
 looking at the invoice sees the full account number (and even GDPR can say a
 word about that), so that's the reason behind the several options.
 
+This module adds "Payment mode" to Invoices Analysis.
```
Comment (pedrobaeza) [inline account_payment_partner/README.rst]: Put this in the list above (and you can take the occasion to correct the typo: `severals` > `several`

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please move it to v14

**Pull request merged**